src/Shell/Task/ModelTask.php

```diff
@@ -113,6 +113,16 @@ public function bake($name)
         $data = $this->getTableContext($tableObject, $table, $name);
         $this->bakeTable($tableObject, $data);
         $this->bakeEntity($tableObject, $data);
+        if (array_key_exists('connection', $this->params) &&
```

Comment:
Why do we need array_key_exists?
Shouldnt

    if ($this->param('connection') && ...)

suffice?

Reply:
@dereuromark Mark you are correct. When I wrote that code I was unaware that $this->param() could be used as you propose. array_key_exists() should be changed. I'm on the road at present but can edit the PR by Monday.

Reply:
@dereuromark Mark, thanks for the ping. I see I missed that we need test coverage here. I'll see what I can do on that, but will be at least Monday 2018-08-20 before I can get to it.